Makefile

```diff
@@ -208,6 +211,12 @@ check-generate:
 check: $(GOIMPORTS) $(GOLANGCI_LINT) $(HELM) $(IMPORT_BOSS)
 	@hack/check.sh --golangci-lint-config=./.golangci.yaml ./cmd/... ./extensions/... ./pkg/... ./plugin/... ./test/...
 	@hack/check-imports.sh ./charts/... ./cmd/... ./extensions/... ./landscaper/... ./pkg/... ./plugin/... ./test/... ./third_party/...
+
+	@echo "> Check $(LOGCHECK_DIR)"
+	@cd $(LOGCHECK_DIR); $(abspath $(GOLANGCI_LINT)) run -c $(REPO_ROOT)/.golangci.yaml --timeout 10m ./...
+	@cd $(LOGCHECK_DIR); go vet ./...
+	@cd $(LOGCHECK_DIR); $(abspath $(GOIMPORTS)) -l .
```

Comment:
Why isn't this checked "normally" (i.e., with an additional parameter to the `check.sh` invocation (same below))?

Reply:
Because logcheck is in its own go module, so you have to `cd` into it for the `go` commands etc. to work.